sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -348,23 +348,29 @@ public new bool IsRunning
         public new string Identifier => base.Identifier;
 
         /// <summary>
-        ///   The instance of <see cref="EventProcessorClientEventSource" /> which can be mocked for testing.
+        ///   Indicates whether or not this processor should instrument batch event processing calls with distributed tracing.
         /// </summary>
         ///
-        internal EventProcessorClientEventSource Logger { get; set; } = EventProcessorClientEventSource.Log;
+        protected override bool? IsBatchTracingEnabled { get => false; }
```

Comment:
Can we please move this down ~L362, after the internals before the private?

Reply:
it's protected, not private. I'm fine either way though

Reply:
I've been organizing in the order  public, internal, protected, private.    

Up to you.   You've been very gracious with all my formatting nits, I've got no problem if you ignore this one and leave it for me later.